stories/Molecules/Text/Faq/Faq.stories.mdx

```diff
@@ -0,0 +1,73 @@
+    case 'burmese': const burmeseText = {header: 'အပိုင်းခေါင်းစီး', country: 'အယ်လ်ဂျီးရီးယား', description: 'Dummy စာသားသည် ၀ က်ဘ်ဆိုက်၏လှောင်ပြောင်မှုကိုဖြည့်ရန်အသုံးပြုသောအကြောင်းအရာအနည်းငယ်ကိုရည်ညွှန်းသည်။ ဤစာသားသည် ၀ က်ဘ်ဒီဇိုင်နာများအနေဖြင့် ၀ က်ဘ်ဆိုက်သည်အချောထည်ထုတ်ကုန်တစ်ခုအဖြစ်မည်သို့ပုံဖော်ရန်စိတ်ကူးကောင်းမြင်နိုင်စေသည်။ dummy စာသားသည်မည်သည့်အဓိပ္ပါယ်မှမရှိကြောင်းနားလည်ရန်အရေးကြီးသည်။ ၎င်း၏တစ်ခုတည်းသောရည်ရွယ်ချက်မှာမူပိုင်ခွင့်ချိုးဖောက်မှုများကိုမပြုလုပ်ဘဲ“ စကားလုံးကဲ့သို့” အကြောင်းအရာဖြင့်ဖြည့်ရန်ဖြစ်သည်။'}; return burmeseText;
+    case 'japanese': const japaneseText = {header: 'セクションヘッダー', country: 'アルジェリア', description: 'ダミーテキストとは、Webサイトのモックアップを埋めるために使用されるコンテンツの一部を指します。このテキストは、WebデザイナーがWebサイトが完成品としてどのように見えるかをよりよく想像するのに役立ちます。ダミーテキストには何の意味もないことを理解することが重要です。その唯一の目的は、著作権を侵害することなく、「単語のような」コンテンツで空白を埋めることです。'}; return japaneseText;
+    default:
+      return {header: 'Section Header', country: 'Algeria', description: 'Dummy text refers to the bits of content that are used to fill a website mock-up. This text helps web designers better envision how the website will look as a finished product. It is important to understand that dummy text has no meaning whatsoever. Its sole purpose is to fill out blank spaces with “word-like” content, without making any copyright infringements.'};
+  }
+};
+
+<Meta title="Molecules/Text/Faq" component={Faq} />
+
+<style>{`
+  .sbdocs-h1 {
+    font-weight: 700;
+  }
+  .subheading {
+    font-weight: 500;
+    font-size: 22px;
+    color: #161616;
+    letter-spacing: 1.5px;
+    line-height: 24px;
+    margin-bottom: 12px;
+    margin-top: 40px;
+  }
+  .high {
+    font-weight: 500;
+    font-size: 14px;
+    color: #161616;
+    letter-spacing: 1.5px;
+    line-height: 24px;
+    margin-bottom: 12px;
+    margin-top: 40px;
+  }
+  .docblock-code-toggle {
+    display: none;
+  }
+`}</style>
+
+# Faq Component
+The FAQ component is responsible for managing the isOpen and toggle functionality to each Item component define in this component, for display the data used items fileds - Header title, Description.  <code>H5</code> <code>P</code> Atoms tags are used for creating this section.
+
+<Story name="Faq">
+  {(args, { globals: { locale } }) => {
+    const caption = getCaptionForLocale(locale);
+    return <Faq headerText={caption.header} descriptionText={caption.description}></Faq>;
+  }}
+</Story>
+
+<div className="subheading">Dependency</div>
+
+Libraries or another dependency which are used in the component.
+
+<ul>
+<li>For toggle functionality its depend on the jQuery plugin library, that is placed in preview-head.html file.</li>
+<li>Custom JS file (Faq.js) is located in the same component folder.</li>
+<li>ID - (#accordion) is used in the JSX file to connect the Custom JS file to achive the functionality.</li>
```

Comment:
@harishkumar-srijan correct the achieve spelling

Reply:
Done